### DISCUSSION TITLE
Miyoo Mini+ Black screen but audio

### DISCUSSION THREAD

**Discussion by olivershooter** [OnionUI/Onion#1173] [Support] [answered]
This didn't happen during a firmware update or anything. 

Installed OnionOS and have been playing fine for about two weeks. Tried to have a go on Chrono Trigger, did the usual turn off and turn on over a few hours of gameplay while I did chores. Came back and it now boots to a black screen with very loud audio. I see the Miyoo boot logo. 

Before re-flashing the sd card and doing all that - is there a way to delete the memory so Onion doesn't try to boot into the games location/state? Sorry if that doesn't quite make sense, essentially I'm just looking to turn the Miyoo on and it starts at the OnionOS home screen vs remembering where I was in the game. 

Ta.

Edit: booting without sdcard it goes straight to the default Miyoo OS so hardware is fine I believe.

**Comment by Aemiii91** [comment, marked as answer]
Hold MENU while turning on, then it will boot to menu.

**Reply by olivershooter** [reply]
Exactly what I was looking for. Unfortuately, didn't work. So I'm now reflashing the drive.

**Reply by TotallyNotSethP** [reply]
Hey having the exact same issue after playing pico 8 (the standalone version), and this suggestion didn't fix it. Hopefully this can be patched soon...

**Reply by trung-hn** [reply]
I got the same issue with Goodboy Galaxy and this fixes it for me. Thanks!

**Reply by BoWalka** [reply]
worked for me thx

**Comment by courgi** [comment]
> Hey having the exact same issue after playing pico 8 (the standalone version), and this suggestion didn't fix it. Hopefully this can be patched soon...

It does work. You are likely not holding the menu button down long enough. Hold MENU while turning on, keep MENU button held until you see the GameSwitcher, then delete offending game.

I also get Fake-08 rom crashes and above brings the system back every time.

**Comment by slrgt** [comment]
Still happening and the fix isn’t working. Happened with Micro Mages game
